### DISCUSSION TITLE
unsupported Metadata

### DISCUSSION THREAD

**Discussion by Baryonae** [vercel/next.js#60001] [Help]
my terminal shows this log of error 
![image](https://github.com/vercel/next.js/assets/73114771/d38dbb98-52d0-497a-980c-f5f55cc63165)
but I don't really remember adding something like theme color to the metadata before causing this error 
what can the error possibly be for?
```
⚠ Unsupported metadata themeColor is configured in metadata export. Please move it to viewport export instead.
 ⚠ Unsupported metadata themeColor is configured in metadata export. Please move it to viewport export instead.
 ⚠ Unsupported metadata themeColor is configured in metadata export. Please move it to viewport export instead.
 ⚠ Unsupported metadata themeColor is configured in metadata export. Please move it to viewport export instead.
 ⚠ Unsupported metadata themeColor is configured in metadata export. Please move it to viewport export instead.

```

### Example

https://github.com/vercel/next.js/issues/57680

**Comment by ananuness** [comment]
Hii! You can add this viewport object to your project:

```js
export const viewport: Viewport = {
  themeColor: '#colorHere',
}
```

But maybe you will have the same ts error mentioned on the issue #57680 (it's not solved yet).

**Reply by amansaurabh** [reply]
Where to add?

**Reply by ananuness** [reply]
Oh, sorry, in the `RootLayout`
